index.html

```diff
@@ -0,0 +1,317 @@
+  <dt markdown>`version: "v1.0"`</dt>
+  <dd markdown>The [version](#versioning) of the spec against which the document was authored. Processors **must** select an implementation of the spec which can [satisfy](#satisfaction) the specified version.</dd>
+</dl>
+
+The version specifier **must** satisfy the [BNF grammar for `‹version specifier›`](#grammar).
+
+### Ignore meaningless URL components
+
+When extracting the URL's `prefix` and `version`, processors **must** ignore any url components which are not assigned a meaning. This spec assigns meaning to the final two components of the [path](https://tools.ietf.org/html/rfc3986#section-3.3). Other URL components—particularly query strings and fragments, if present—**must** be ignored for the purposes of extracting the `prefix` and `version`.
+
+<figure>
+  <figcaption>
+    <a name=spec-url-ignore-meaningless href=#spec-url-ignore-meaningless class=a-header>Ignoring meaningless parts of a URL</a>
+  </figcaption>  
+  <code class=anatomy>
+    <span class=pink style='--depth: 2'>https://example.com/<span>exampleSpec<aside>prefix</aside></span><aside>
+      identity
+    </aside></span>/<span style='--depth: 2' class=green>v1.0<aside>version</aside></span><span class=grey>?key=val&k2=v2#fragment<aside>ignored</aside></span>
+  </code>
+</figure>
+
+### Why is versioning in the URL, not a directive argument?
+
+The version is in the URL because when a human reader visits the URL, we would like them to be taken to the documentation for the *version of the specification in use by this document*. Many text editors will turn URLs into hyperlinks, and it's highly desirable that clicking the link takes the user to the correct version of the docs. Putting the version information in a separate argument to the `@using` directive would require this.
+
+## Prefixing
+Specifications meant to be referenced via `@using` **should** prefix any schema elements they introduce. The prefix **must** be a string of characters valid within GraphQL typenames, with the exception of `"_"`.
+
+Specs **should** define any new schema elements with names that start with `prefix_`.
+
+```graphql
+#::[Using the `@using` directive with a spec's default prefix]
+schema @using(spec: "https://spec.example.com/example/v1") {
+  query: Query
+}
+
+type User {
+  name: String @example_someDirective
+               # prefixed directive from the example spec
+}
+```
+
+The language of the spec **should** specify the default prefix and **should** identify locations where the prefix is used. The prefix **should not** be elided within documentation; definitions of schema elements provided within the spec **must** include the default prefix.
+
+Specs should attach their prefix to:
+- the names of any types, interfaces, unions, enums, or inputs defined in the spec
```

Comment:
Hmm, this is a list of SDL keywords rather than kinds of named types. I would remove "types", replace "inputs" with "input objects", and include "object types" and "input object types".

Reply:
oh, oops

Reply:
fixed

Reply:
fixed

Reply:
Also missing scalars, which I totally meant to include yesterday :)